src/applications/personalization/profile-2/components/AccountSecurityContent.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+import recordEvent from 'platform/monitoring/record-event';
+import {
+  isLOA3 as isLOA3Selector,
+  isMultifactorEnabled as isMultifactorEnabledSelector,
+  selectProfile,
+} from 'platform/user/selectors';
+import { ssoe as ssoeSelector } from 'platform/user/authentication/selectors';
+
+import ProfileInfoTable from './ProfileInfoTable';
+import TwoFactorAuthorizationStatus from './TwoFactorAuthorizationStatus';
+import IdentityVerificationStatus from './IdentityVerificationStatus';
+import MHVTermsAndConditionsStatus from './MHVTermsAndConditionsStatus';
+
+export const AccountSecurityContent = ({
+  isIdentityVerified,
+  isMultifactorEnabled,
+  mhvAccount,
+  showMHVTermsAndConditions,
+  useSSOe,
+}) => {
+  const data = [
+    {
+      title: 'Identity verification',
+      value: (
+        <IdentityVerificationStatus isIdentityVerified={isIdentityVerified} />
+      ),
+    },
+    {
+      title: '2-factor authentication',
+      value: (
+        <TwoFactorAuthorizationStatus
+          isMultifactorEnabled={isMultifactorEnabled}
+          useSSOe={useSSOe}
+        />
+      ),
+    },
+  ];
+
+  if (showMHVTermsAndConditions) {
+    data.push({
+      title: 'Terms and conditions',
+      value: <MHVTermsAndConditionsStatus mhvAccount={mhvAccount} />,
+    });
+  }
```

Comment:
This is cool. Yeah I see what you were saying about just keeping the static method on `MHVTermsAndConditionsStatus`. Makes sense to me.